src/AzureStack/Admin/BackupAdmin/Backup.Admin/Microsoft.AzureStack.Management.Backup.Admin.csproj

```diff
@@ -7,7 +7,7 @@
     <PropertyGroup>
         <PackageId>Microsoft.AzureStack.Management.Backup.Admin</PackageId>
         <Description>Provides developers with libraries for the Backup admin under AzureStack to manage backups.</Description>
-        <VersionPrefix>0.1.0-preview</VersionPrefix>
+        <VersionPrefix>0.1.1-preview</VersionPrefix>
         <AssemblyName>Microsoft.AzureStack.Management.Backup.Admin</AssemblyName>
         <PackageTags>Microsoft Azure Stack;Backup;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
         <PackageReleaseNotes>
```

Comment:
Please update `PackageReleaseNotes`
Changing return types is a breaking change, please update version to `0.2.0-preview`